apps/builder/app/canvas/custom-components/link.tsx

```diff
@@ -46,7 +46,11 @@ export const preserveBuildParams = (href: string, sourceSearch: string) => {
   return `${url.pathname}${search === "" ? "" : `?${search}`}`;
 };
 
-type Props = Omit<ComponentProps<"a">, "href"> & { href?: string };
+type Props = Omit<ComponentProps<"a">, "href" | "target"> & {
+  href?: string;
+  target?: "self" | "blank" | "parent" | "top";
+  preload?: "none" | "prefetch" | "prerender";
```

Comment:
What is this `preload` prop? 

Remix has [prefetch](https://remix.run/docs/en/v1/components/link#prefetch) which you already have below. 

And `<a>` doesn't have a `preload` attribute https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a

Reply:
that's me having committed intermediate change and forgotten.
The question is how we ended up not having any errors there? Why are all the link props type copy pasted across the places?